great_expectations/validator/validator.py

```diff
@@ -691,6 +697,15 @@ def active_batch(self):
         active_batch = self.batches.get(active_batch_id) if active_batch_id else None
         return active_batch
 
+    def set_active_batch_id(self, batch_id):
```

Comment:
@talagluck I may be missing some existing context, but could we not define the `active_batch` (or `active_batch_id`) property (or properties if both, not sure) and use the standardized `@property` and `setter` decorators on it (or on them)?  Thanks.